packages/react-components/react-motion-components-preview/library/src/components/Fade/Fade.test.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import { Fade, FadeExaggerated, FadeSnappy } from './Fade';
+import { render, waitFor } from '@testing-library/react';
+import { motionTokens } from '@fluentui/react-motion';
+import { mockAnimation } from '../../testing/testUtils';
+
+describe('Fade motion component', () => {
+  let animateSpy: jest.SpyInstance;
+  const testElement = <div data-testid="fade-content">Test</div>;
+
+  beforeAll(() => {
+    HTMLElement.prototype.animate = () => mockAnimation;
+  });
+
+  beforeEach(() => {
+    animateSpy = jest.spyOn(HTMLElement.prototype, 'animate').mockImplementation(() => mockAnimation);
```

Comment:
```suggestion
    animateSpy = jest.spyOn(HTMLElement.prototype, 'animate');
```
This works without referencing `mockAnimation` a second time.